src/main/groovy/com/athaydes/gradle/ceylon/CeylonConfig.groovy

```diff
@@ -20,7 +20,7 @@ class CeylonConfig {
     Boolean verbose = false
     String javaRuntimeDestination
     String entryPoint
-    String jarOutput
+    String moduleVersion
```

Comment:
I am removing this because there's already a version in the Ceylon module file (which we use)... there's even a version in the Gradle build, which we don't really use for anything... I guess we could use that instead of this if the Ceylon module version is not correct for whatever reason, but for now we don't seem to need this.